test/active_model_serializers/logging_test.rb

```diff
@@ -72,6 +72,11 @@ def test_logs_the_duration
         ActiveModelSerializers::SerializableResource.new(@post).serializable_hash
         assert_match(/\(\d+\.\d+ms\)/, @logger.messages)
       end
+
+      def test_logs_without_serializer
```

Comment:
should be logs nothing when serializer is not found (i.e. is null)

and then assert that the serializable resource serializer is the Null serializer